### PR TITLE
Earn Section: Update landing page

### LINKED ISSUE #34517: Earn Section: Code up copy variations for each card
## Description

We'll need to instrument different copy and links depending on the plan that a customer is currently on:

copy

### PR DESCRIPTION
**Pull request by pablinos**
This is is the main landing page for the Earn section using the new card layout. It's still WIP and this is a skeleton version with a couple of cards, but it will form the basis of the section as a whole.

#### Testing instructions

Use Calypso live, and go to the `/earn/` you should be presented with the new card layout. Only the recurring payments button currently works.

Fixes #34517 

Some of the bits left to do are:

- [x] Update SVGs
- [x] Include the other cards and copy
- [x] Increase the default size of the main div (it looks a bit narrow)
- [x] Add the click handlers for the CTA buttons on each card, both for upgrading and to go to the appropriate section
- [ ] Add the analytics
- [x] Find the most appropriate way of not having a Learn More link on Simple Payments
- [ ] Work out the best way to detect the activation of each feature
- [ ] Provide the 3rd variation of each card when the feature is activated

### CODE REVIEW

**Comment by eliorivero** [comment]
In https://github.com/Automattic/wp-calypso/pull/34896/commits/790599f82e85caec3583c2f6da0da7ff5a3a7098

Ensure the action panel cta is anchored to bottom to avoid having the cta elements at different height when two or more action panels with different content height are located side by side

### Before 
Captura de Pantalla 2019-07-25 a la(s) 19 41 03

### After
Captura de Pantalla 2019-07-25 a la(s) 19 41 11

**Comment by eliorivero** [comment]
In https://github.com/Automattic/wp-calypso/pull/34896/commits/5af61e73b11a9dd0e8a8853367220c7eb8dff515

Add the learnMoreLink prop to PromoSection and pass it to its PromoCardCta so we can set a Learn more link from the Earn home.

### Before
Captura de Pantalla 2019-07-25 a la(s) 22 40 21

### After
Captura de Pantalla 2019-07-25 a la(s) 22 40 09

File: client/my-sites/earn/home.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+					components: {
+						em: <em />,
+					},
+				}
+			),
+			image: {
+				path: '/calypso/images/earn/woo.svg',
+			},
+			cta: {
+				text: translate( 'Upgrade to Premium Plan' ),
```
Comment (a8ci18n) [inline client/my-sites/earn/home.tsx]: Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Upgrade to a Premium Plan!' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/earn/home.tsx [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+					},
+				}
+			),
+			image: {
+				path: '/calypso/images/earn/simple-payments.svg',
+			},
+			cta: {
+				feature: FEATURE_SIMPLE_PAYMENTS,
+				upgradeButton: {
+					text: translate( 'Upgrade to Premium Plan' ),
```
Comment (a8ci18n) [inline client/my-sites/earn/home.tsx]: Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Upgrade to a Premium Plan!' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/earn/home.tsx [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+					action: () => page( `/earn/ads-earnings/${ selectedSiteSlug }` ),
+				},
+			},
+			learnMoreLink: 'https://wordads.co/',
+		};
+	};
+
+	const promos: PromoSectionProps = {
+		header: {
+			title: translate( 'Start earning money' ),
```
Comment (a8ci18n) [inline client/my-sites/earn/home.tsx]: Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'Start earning money now' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Comment (pablinos) [reply client/my-sites/earn/home.tsx]: @davemart-in Do you think we should add 'now' into the title of the header component?

File: client/my-sites/earn/home.tsx [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+	const getSimplePaymentsCard = () => {
+		const supportLink =
+			'https://en.support.wordpress.com/wordpress-editor/blocks/simple-payments-block/';
+		const cta = hasSimplePayments
+			? {
+					text: translate( 'Collect One-time Payments' ),
+					action: supportLink,
+			  }
+			: {
+					text: translate( 'Upgrade to Premium Plan' ),
```
Comment (a8ci18n) [inline client/my-sites/earn/home.tsx]: Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Upgrade to a Premium Plan!' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/earn/home.tsx (hunk 3 above)
Comment (davemart-in) [reply client/my-sites/earn/home.tsx]: That's fine I think. 👍

File: client/components/promo-section/index.tsx [hunk 5]
```diff
@@ -12,6 +12,7 @@ import PromoCardCta, { Props as PromoCardCtaProps } from './promo-card/cta';
 interface PromoSectionCardProps extends PromoCardProps {
 	body: string;
 	cta?: PromoCardCtaProps;
+	learnMoreLink?: PromoCardCtaProps;
```
Comment (sirreal) [inline client/components/promo-section/index.tsx]: Is there a mistake here?

```suggestion
	learnMoreLink: PromoCardCtaProps['learnMoreLink'];
```

File: client/components/promo-section/index.tsx [hunk 6]
```diff
@@ -12,6 +12,7 @@ import PromoCardCta, { Props as PromoCardCtaProps } from './promo-card/cta';
 interface PromoSectionCardProps extends PromoCardProps {
 	body: string;
 	cta?: PromoCardCtaProps;
```
Comment (sirreal) [inline client/components/promo-section/index.tsx]: Is there a mistake here?

```suggestion
	cta: PromoCardCtaProps['cta'];
```

**Review by sirreal — COMMENTED** [review]
A couple of notes on types.

File: client/components/promo-section/index.tsx (hunk 5 above)
Comment (pablinos) [reply client/components/promo-section/index.tsx]: You're right the types got a bit confused here. I've updated them, although I'm not sure if there is a better property name to use than `ctaProps`. I'd be interested to know if you think this is better.

**Comment by pablinos** [comment]
I've updated the types after the changes we made for analytics and the suggestions from @sirreal . I'd be interested to hear whether you both (@sirreal & @eliorivero) think it's ok now.

**Comment by eliorivero** [comment]
`ctaProps` could be named `actions`, to avoid having a prop for something already named like that, for example:

```
ctaProps: {
	cta,
},
```

**Pull request merged**